app/questionnaire/value_source_resolver.py

```diff
@@ -75,6 +75,23 @@ def _get_answer_value(
         ):
             return answer.value
 
+    def _resolve_list_item_id_for_answer(self, answer_id: str) -> str | None:
```

Comment:
Fixed the function name 👍 I was thinking the answer is what I'd consider repeating rather than the id, and thought the docstring might be too verbose if I added "and the answer id is for a repeating answer" but up for tweaking it or adding if that makes it easier to follow